src/main/java/org/broadinstitute/consent/http/resources/Resource.java

```diff
@@ -71,6 +72,18 @@ StreamingOutput createStreamingOutput(InputStream inputStream) {
         };
     }
 
+    void validateFileDetails(FormDataContentDisposition fileDetail) {
+        FileValidator validator = new FileValidator();
+        boolean validName = validator.isValidFileName("validating uploaded file name", fileDetail.getFileName(), true);
```

Comment:
So does the validator ensure that the filename follows a uuid format?

Reply:
No, it uses a library regex. You can dig into the code here: https://github.com/augustd/owasp-java-fileio/blob/b2f13dbc0002cc6c2ffe880f76211df90e7aa4d5/src/main/java/org/owasp/fileio/FileValidator.java#L35

Reply:
In this code, we're not trying to force the file into a UUID format. We only want to validate it for OWASP reasons, and that's it. For our purposes, we really don't care what the file name is. We store that separately as a raw string on the DAR and not as part of the actual file.